apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/ProjectsResourceTest.java

```diff
@@ -1592,10 +1592,14 @@ void update__whenNameIsBlank__thenRejectTheUpdate() {
     @TestInstance(TestInstance.Lifecycle.PER_CLASS)
     class DeleteProject {
 
-        @Test
+        @ParameterizedTest
+        @MethodSource
         @DisplayName("Success")
-        void delete() {
-            var id = createProject(factory.manufacturePojo(Project.class).toBuilder().build());
+        void delete(String projectName) {
```

Comment:
I suppose he meant that it will be recreated if needed later, you could refer to his comment:
https://comet-ml.atlassian.net/browse/OPIK-451?focusedCommentId=39976

Reply:
Great